test/viewmodels/ImportVatViewModelSpec.scala

```diff
@@ -0,0 +1,384 @@
+
+  private def shouldContainCorrectBackLink(viewModel: ImportVatViewModel)(implicit config: AppConfig): Assertion =
+    viewModel.backLink mustBe Some(config.customsFinancialsFrontendHomepage)
+
+  private def shouldContainCorrectHeading(viewModel: ImportVatViewModel)(implicit msgs: Messages): Assertion =
+    viewModel.heading mustBe h1Component(
+      "cf.account.vat.title",
+      id = Some("import-vat-certificates-heading"),
+      classes = "govuk-heading-xl"
+    )
+
+  private def shouldContainCorrectCertificateAvailableGuidance(viewModel: ImportVatViewModel)(implicit
+    msgs: Messages
+  ): Assertion =
+    viewModel.certificateAvailableGuidance mustBe pComponent("cf.account.vat.available-text")
+
+  private def shouldContainCorrectLast6MonthsHeading(viewModel: ImportVatViewModel)(implicit
+    msgs: Messages
+  ): Assertion =
+    viewModel.last6MonthsH2Heading mustBe h2Component("cf.account.vat.your-certificates.heading")
+
+  private def shouldContainNotificationPanel(viewModel: ImportVatViewModel) = viewModel.notificationPanel mustBe empty
+
+  private def shouldNotContainNotificationPanel(viewModel: ImportVatViewModel) =
+    viewModel.notificationPanel mustBe empty
+
+  private def shouldContainCurrentStatements(
+    viewModel: ImportVatViewModel,
+    certificatesForAllEoris: Seq[VatCertificatesForEori]
+  )(implicit msgs: Messages): Assertion =
+    viewModel.currentStatements mustBe expectedImportVatCurrentStatements(certificatesForAllEoris)
+
+  private def shouldNotContainCurrentStatements(viewModel: ImportVatViewModel): Assertion =
+    viewModel.currentStatements mustBe Seq.empty
+
+  private def shouldNotContainCurrentStatementsNotAvailableGuidance(viewModel: ImportVatViewModel): Assertion =
+    viewModel.currentStatementsNotAvailableGuidance mustBe empty
+
+  private def shouldContainCurrentStatementsNotAvailableGuidance(
+    viewModel: ImportVatViewModel
+  )(implicit msgs: Messages): Assertion =
+    viewModel.currentStatementsNotAvailableGuidance mustBe Some(
+      pComponent(
+        "cf.account.vat.no-certificates-available",
+        id = Some("no-certificates-available-text")
+      )
+    )
+
+  private def shouldContainCorrectCertsOlderThan6MonthsGuidance(viewModel: ImportVatViewModel)(implicit
+    msgs: Messages
+  ) =
+    viewModel.certsOlderThan6MonthsGuidance mustBe GuidanceRowWithParagraph(
+      h2Component(
+        "cf.account.vat.older-certificates.heading",
+        id = Some("missing-certificates-guidance-heading"),
+        classes = "govuk-heading-m govuk-!-margin-top-9"
+      ),
+      Some(
+        linkComponent(
+          "cf.account.vat.older-certificates.description.link",
+          location = URL_TEST,
+          preLinkMessage = Some("cf.account.vat.older-certificates.description.1")
+        )
+      )
+    )
+
+  private def shouldContainCorrectChiefDeclarationGuidance(viewModel: ImportVatViewModel)(implicit
+    appConfig: AppConfig,
+    msgs: Messages
+  ) =
```

Comment:
should these refs end with ) = on there own line?

Reply:
Formatting is done by scala formatter